app/views/RoomView/RightButtons.tsx

```diff
@@ -346,6 +380,12 @@ class RightButtonsContainer extends Component<IRightButtonsProps, IRigthButtonsS
 		}
 		return (
 			<HeaderButton.Container>
+				<HeaderButton.Item
+					color={notificationIsEnabled ? themes[theme].headerTintColor : themes[theme].fontDanger}
```

Comment:
use `withTheme` and use `colors`

Reply:
Why? Isn't the best way to keep the same pattern for LeftButtons and RightButtons? Then when we decide to refactor both components, the developer can understand the expected behavior.

Reply:
Not if the pattern is wrong.
The right thing to do is to tidy things up little by little as you move, if you are adding something new, correct the mistakes of the past.
Like migrating to hooks, something you did in the audio player.